drake/examples/Pendulum/Pendulum.h

```diff
@@ -2,44 +2,77 @@
 #define _PENDULUM_H_
 
 #include <iostream>
+#include <cmath>
 #include "DrakeSystem.h"
-
+#include "LCMCoordinateFrame.h"
+#include "BotVisualizer.h"
 
 class Pendulum : public DrakeSystem {
 public:
-  Pendulum(void) :
+  Pendulum(const std::shared_ptr<lcm::LCM>& lcm) :
```

Comment:
and we can't do const lcm::LCM& because the lcm methods that we need to call are not const.